apps/api/src/middleware/openapi-meta.ts

```diff
@@ -52,6 +57,11 @@ export const openapiMeta: OpenAPIObjectConfigure<{ Bindings: Env }, string> = {
       description:
         "Present and past LARC (https://larc.uci.edu/) sections. Sourced from LARC's enrollment site (https://enroll.larc.uci.edu/).",
     },
+    {
+      name: "Study Rooms",
+      description:
+        "Study room data, such as location, directions, capacity, and availability. Sourced from UCI Libraries' reservation site (https://spaces.lib.uci.edu/).",
```

Comment:
This list of actual datapoints may be too fine for a category descriptor. I think that list should be removed.